src/stories/AlbumTumbnail.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import AlbumThumbnail from '@/domain/_common/components/AlbumThumbnail';
+
+const meta = {
+  title: 'Components/AlbumThumbnail/AlbumThumbnail',
```

Comment:
```suggestion
  title: 'Components/AlbumThumbnail',
```
이중폴더 구조가 됐는데 의도된건가요?